WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,49 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
+
+## Global package:
+```
+npm uninstall -g angular-cli
+npm cache clean
+npm install -g angular-cli@webpack
+```
+
+## Local project package:
+```
+rm -rf node_modules dist tmp typings
+npm install --save-dev angular-cli@webpack
```

Comment:
Hans said that the release would not be made as `1.0.0-beta.11` but rather as `webpack`, so we need to install that particular version for now.